### ISSUE TITLE
Inserting media, WP 5.2 classic editor

### ISSUE THREAD

**Issue by msassmann** [SpiderLabs/owasp-modsecurity-crs#1452]
As @lifeforms suggested, I'm opening a new issue for this.
Using WP 5.2.1 Rule 941160 is causing a false positive when inserting media into the wp classic editor. Wordpress Exceptions are enabled, using OWASP 3.1.

[modsec_audit.log](https://github.com/SpiderLabs/owasp-modsecurity-crs/files/3293189/modsec_audit.log)

**Comment by theMiddleBlue** [comment]
Thanks @msassmann!

In this case, the exclusion rule 9002770 should remove `ARGS:html` from all rules, but it does only if you're using ModSecurity v2. If you're using libmodsecurity (v3.0) your problem should be related to https://github.com/SpiderLabs/ModSecurity/issues/2110. We're still waiting for this PR https://github.com/SpiderLabs/ModSecurity/pull/2114 😞

**Comment by diablodale** [comment]
Still waiting on PR dependency

**Comment by lifeforms** [comment]
This problem should have been fixed in CRS 3.2.0 by working around the ModSec issues theMiddleBlue described. Rule 9002770 should catch the problem correctly now.

If you are still having problems after the upgrade, please open a new issue with the log!
